internal/resources/metal/organization/resource_test.go

```diff
@@ -98,8 +81,68 @@ func TestAccMetalOrganization_importBasic(t *testing.T) {
 	})
 }
 
+// Test to verify that switching from SDKv2 to the Framework has not affected provider's behavior
+func TestAccMetalOrganization_upgradeFromVersion(t *testing.T) {
```

Comment:
This test is failing; it looks like there are some differences between the SDK and framework implementations:
```
=== NAME  TestAccMetalOrganization_upgradeFromVersion
    resource_test.go:88: Step 2/2 error: Pre-apply plan check(s) failed:
        expected empty plan, but equinix_metal_organization.test has planned action(s): [update]
```

Reply:
This test is still failing in the latest CI, so it looks like there is still a difference between how the SDKv2 resource stored organizations in state and how the framework resource stores organizations in state.  I think this is a test that can be run locally against the playground org.  I'm not sure how to get more information out of the test, though.  @ocobles do you know if there's a way to get terraform acceptance tests to print the plan for each step?